src/features/DebugSession.ts

```diff
@@ -440,6 +458,10 @@ export class DebugSessionFeature
             return PREVENT_DEBUG_START_AND_OPEN_DEBUGCONFIG;
         }
 
+        if (config.sessionName) {
+            config.createTemporaryIntegratedConsole = true;
```

Comment:
My only thought is that we tried to move away from "integrated console" to "extension terminal" but I am unsure how much of that name change made it to this particular feature.

Reply:
I can probably fix that up, stay tuned.

Reply:
So I can either fix it in this commit or we commit this and I submit another PR to rename/alias createTemporaryIntegratedConsole to createTemporaryPowershellSession. Which would you prefer?

Reply:
Seeing as this was merged I will address it in separate PR.

Reply:
Oh yeah I didn't see it was set to auto complete.